apps/client/src/dashboard/dashboard.page-crumbs.ts

```diff
@@ -0,0 +1,7 @@
+import type { PageCrumb } from '../breadcrumbs/page-crumb.type';
+
+export const DASHBOARD_PAGE_CRUMBS: PageCrumb[] = [
+  { text: 'Dashboards', href: '/dashboards' },
+  // TODO: Replace with real dashboard name once resource is being loaded
+  { text: 'Dashboard Name', href: 'no-op' }, // no link needed for deepest resource
+];
```

Comment:
does the route provide us enough info to generate the crumbs? it would be less static configs and conventions to do generation dynamically.

Reply:
The primary problem is /dashboards/123 is not a child route of /dashboards. There are only two route matches: / and /dashboards/123. I was not able to figure out a way to safely assume something like `home > dashboards > dashboard name` from only two matches. It's too brittle of an assumption.